### DISCUSSION TITLE
support for wildcards?

### DISCUSSION THREAD

**Discussion by mtooke** [rdilweb/docusaurus-plugin-remote-content#57] [Q&A]
Hi,

Two quick questions that I wasn't able to find an answer to:

1. Can you use wildcards to bring whole directories of .md files over? 
2. Can you bring the sidebars file in as well, or do you just maintain that in the destination repo?

Thanks.

**Comment by RDIL** [comment]
Hi,
1. It depends on where these files are stored.  More than likely, you can, but you need to actually collect a list of files, just like how [this](https://github.com/rdilweb/docusaurus-plugin-remote-content/issues/37#issuecomment-1131494910) library works.
2. Probably, it should just be the same as fetching any other text file, I think.
